playground/backend/cmd/server/controller.go

```diff
@@ -104,6 +103,23 @@ func (controller *playgroundController) GetRunOutput(ctx context.Context, info *
 	return &pipelineResult, nil
 }
 
+// GetRunError is returning error output of execution for specific pipeline by PipelineUuid
+func (controller *playgroundController) GetRunError(ctx context.Context, info *pb.GetRunErrorRequest) (*pb.GetRunErrorResponse, error) {
```

Comment:
Is there a way we can nicely combine `GetRunOutput` and `GetRunError`? They do practically the same.

Reply:
Add one more method to combine `GetRunOutput()`, `GetCompileOutput()` and `GetRunError()`.

Reply:
On the other hand, let's leave them as is since they literally consist of 5 lines of code (excluding braces of course😁)